cli/running/process.go

```diff
@@ -420,3 +421,58 @@ func (process *Process) Log(follow bool, n int) error {
 
 	return nil
 }
+
+func (process *Process) Clean() ProcessRes {
+	res := ProcessRes{
+		ProcessID: process.ID,
+	}
+
+	pathsToDelete := []string{
+		process.logFile,
+		process.workDir,
+		process.consoleSock,
+		process.notifySockPath,
+		// PID file isn't deleted
+		// since it's used by Cartridge CLI to start and stop instance
+	}
+
+	var existedFiles []string
+	var skipped = true
+
+	for _, path := range pathsToDelete {
```

Comment:
I think it should be splitted into two parts: validation and execution. At first you check possibility to delete files and only then remove it physically. It should prevent partial dropping if you don't have enough right to drop files for e.g. the third instance and delete two first but don't try to clean for fourth and fifth.

Reply:
Now files deletion process isn't stopped, if fails for one file. It just collects errors and returns an error if failed for some files.